proto/desmos/profiles/v1beta1/query.proto

```diff
@@ -151,47 +145,16 @@ message QueryUserBlocksResponse {
 
 // ___________________________________________________________________________________________________________________
 
-// QueryChainLinksRequest is the request type for the Query/ChainLinks RPC
-// endpoint
-message QueryChainLinksRequest {
-  option (gogoproto.equal) = false;
-  option (gogoproto.goproto_getters) = false;
-
-  // pagination defines an optional pagination for the request.
-  cosmos.base.query.v1beta1.PageRequest pagination = 1;
-}
-
-// QueryChainLinksResponse is the response type for the Query/ChainLinks RPC
-// method.
-message QueryChainLinksResponse {
-
-  // chains links represent the list of all the links for all users
-  repeated desmos.profiles.v1beta1.ChainLink chains_links = 1  [ (gogoproto.nullable) = false ];
-
-  cosmos.base.query.v1beta1.PageResponse pagination = 2;
-}
-
-// ___________________________________________________________________________________________________________________
-
-// QueryUserChainLinksRequest is the request type for the Query/UserChainLinks RPC
+// QueryProfileByChainLinkRequest is the request type for the Query/ProfileByLink RPC
 // endpoint
-message QueryUserChainLinksRequest {
-  option (gogoproto.equal) = false;
-  option (gogoproto.goproto_getters) = false;
-
-  // address of the user to query the links for
-  string user = 1;
-
-  // pagination defines an optional pagination for the request.
-  cosmos.base.query.v1beta1.PageRequest pagination = 2;
+message QueryProfileByChainLinkRequest {
+  string chain_name = 1 ;
+  string target = 2 ;
```

Comment:
I think we should rename this `target_address` to make it easier to comprehend what it should contain